lib/queueservice.js

```diff
@@ -107,12 +107,10 @@ export default class TaskQueue {
     let claim;
     try {
       claim = await this.stats.timeGen(
-        'tasks.time.claim',
+        'claimTask',
```

Comment:
I'll be taking a look at those PRs this morning.  What I'm wanting to capture (and probably can with those PRs) is worker level claims/failures/exceptions as well as the time to claim the task (including all retries, etc).  My hopes is that the client can give me that and then I'm definitely all for not including those things here.